### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Bat-Script** [community/community#49896] [Other Feature Feedback, Questions, & Ideas] [answered]
How can I change my profile picture?

**Comment by Silvasly** [comment]
You can just:

1. Click on your Profile in the top right corner of the screen, besides the plus icon.
2. Go down, and find the option "Your profile" then click it.
3. Then click your image.
4. After that click on the edit icon next to your avatar image.
5. Finally, just upload your image

**Comment by bm4578** [comment]
1. Log in to your GitHub account.
2. Click on your profile picture in the top-right corner of the page.
3. Click on the "Settings" option from the drop-down menu.
4. On the left-hand side of the page, click on the "Profile" option.
5. Scroll down to the "Avatar" section.
6. Click on the "Choose your avatar" button.
7. Select the image you want to use as your profile picture.
8. Crop the image if necessary using the cropping tool.
9. Click on the "Upload new image" button.

**Comment by neutrinomuon** [comment, marked as answer]
Here's a quick way to change your profile picture on Github:

1. Go to GitHub and login
2. Find your current profile picture in the top right corner
3. Click it and select "Settings"
<img width="600" alt="Captura de ecrã 2023-03-13, às 20 42 52" src="https://user-images.githubusercontent.com/8226984/224828748-8037d111-f3f5-4bdc-99eb-ab1363c6809b.png">

5. Click "Public Profile" on the left sidebar
6. Click on your pic and hit edit, then choose 'Upload a photo' or 'Revert to avatar'
<img width="600" alt="Captura de ecrã 2023-03-13, às 20 44 24" src="https://user-images.githubusercontent.com/8226984/224828922-bed31bc3-7d26-4f66-977f-a4974063d981.png">

7. Crop it if needed and hit "Save"
8. Celebrate with a victory dance!

<img width="100" src="https://user-images.githubusercontent.com/8226984/224830093-6972246a-883a-4709-be65-db1733e3a9bb.png">

And that's it! You're now the proud owner of a brand new Github profile pic

**Comment by neutrinomuon** [comment]
Please indicate which one of the provided solutions was helpful to other members of the community by marking it as answered! Thanks in advance

**Comment by Bat-Script** [comment]
Thank you
